src/pages/ProfilePage/views/FavoritesView/FavoritesView.tsx

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import Grid from '@mui/material/Grid';
+import { readAll } from 'src/firebase';
+import { useState, useEffect } from 'react';
+import FavoriteHouses from '../../../../components/ProfilePage/FavoriteHouses/FavoriteHouses';
+
+const FavoritesView = () => {
+  const [favorites, setFavorites] = useState<BasicHouseData[]>([]);
+
+  // GET DATA FROM FIREBASE (TRY OUT) - DELETE AFTER SUCCESSFULLY PUTTING DATA ON THE SERVER
+  const fetchHouses = async () => {
+    const fetchedHouses = await readAll('houses');
+    setFavorites(() => fetchedHouses);
+  };
+
+  useEffect(() => {
+    fetchHouses();
+  }, []);
+
+  // AXIOS FETCH DATA FROM SERVER - USE IT WHEN IT ALL WORKS
+  // const url = 'http://localhost:4000/users/my-favorites';
+
+  // useEffect(() => {
+  //   getFavorites();
+  // }, [])
+
+  // const getFavorites = () => {
+  //   axios.get<BasicHouseData[]>(`${url}`)
+  //     .then((res) => {
+  //       const allFavorites = res.data;
+  //       setFavorites(allFavorites);
+  //     })
+  //     .catch((err) => console.error(`Error: ${err}`));
+  // };
+  return (
+    <Grid container spacing={12}>
```

Comment:
Would it be possible for this view to be as tall as the number of the rows of favourites? The gap between the bottom edge of this view and the bottom of the page can be quite significant:
![Screenshot from 2022-04-22 22-08-36](https://user-images.githubusercontent.com/47178494/164786675-2bfbbcc1-da26-4d7d-9fc7-4d0b417da84e.png)

Reply:
Yes, It's strange that u have it closed with scroll cuz I've put different class so It would take the whole space, let me look into it